### ISSUE TITLE
Seeing ^M in lots of files

### ISSUE THREAD

**Issue by ukBaz** [martinohanlon/BlueDot#31]
For example if I look at examples/dot_debug.py using vi I see the following
```
def released():^M
    print("Released: x-{} y-{}".format(dot.position.x, dot.position.y))^M
    print()^M
^M
```

**Comment by martinohanlon** [comment]
Thanks, Ive seen them too, on my list to sort!

**Comment by martinohanlon** [comment]
resolved in 0.0.7 (hopefully i got em all!)

**Comment by ukBaz** [comment]
Saw this appear again in the `.travis.yml`.
Have you got [core.autocrlf](https://git-scm.com/book/tr/v2/Customizing-Git-Git-Configuration) set?
